lib/pmdtester/builders/diff_builder.rb

```diff
@@ -7,25 +7,29 @@ class DiffBuilder
     # http://pmd.sourceforge.net/report_2_0_0.xsd
 
     def build(base_report, patch_report)
-      base_doc = Nokogiri::XML(File.read(base_report))
-      patch_doc = Nokogiri::XML(File.read(patch_report))
+      base_doc = Nokogiri::XML(File.read(base_report)).remove_namespaces!
```

Comment:
What was the problem with namespaces here? Just want to make sure, we don't have something weird in our schema...

Reply:
Don't worry. We don't care about the namespace when parsing the report of PMD, so we can just remove it. Refer to https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri/XML/Document#remove_namespaces%21-instance_method